mars/metric/api.py

```diff
@@ -32,6 +34,18 @@ def init_metrics(config: Dict[str, Any] = None):
     _metric_backend = metric_config.get("backend", "console")
     if _metric_backend not in _backends_cls:
         raise NotImplementedError(f"Do not support metric backend {_metric_backend}")
+    if _metric_backend == "prometheus":
+        conf = metric_config.get("conf", {})
+        port = int(conf.get("port", 0))
+        try:
+            from prometheus_client import start_http_server
+
+            start_http_server(port)
+            logger.info("Finished startup prometheus http server and port is %d", port)
+        except ImportError:
+            logger.info(
+                "Do not startup prometheus http server because there is no prometheus_client"
```

Comment:
Use `logger.warning("Failed to start...")`.

Reply:
Ok. Thanks.